src/containers/products-filters-container/products-filters-container.js

```diff
@@ -0,0 +1,97 @@
+import React, { useMemo } from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import {
+  ListItemText,
+  FormControl,
+  InputLabel,
+  Select,
+  Input
+} from '@material-ui/core';
+import MenuItem from '@material-ui/core/MenuItem';
+import Checkbox from '@material-ui/core/Checkbox';
+import Badge from '@material-ui/core/Badge';
+import { useStyles } from './products-filters-container.styles';
+import { productsTranslations } from '../../translations/product.translations';
+
+const badgePosition = {
+  vertical: 'top',
+  horizontal: 'left'
+};
+
+const { CATEGORIES } = productsTranslations;
+
+const ProductsFilterContainer = ({
+  buttonName,
+  labels,
+  productFilter,
+  list,
+  filterHandler
+}) => {
+  const styles = useStyles();
+  const { filters } = useSelector(({ Products }) => ({
+    filters: Products.filters
+  }));
+
+  const { categoryFilter } = filters;
+
+  const formGroupOptions = list.map(({ _id }, idx) => (
+    <MenuItem key={labels[idx]} value={_id || labels[idx]}>
```

Comment:
It is the same issue as if you use `idx` directly because `labesl[idx]` depends on the `idx`, please, use some unique value from the form object but without depending on the `idx`

Reply:
fixed